### PR TITLE
Script step

### PR DESCRIPTION
**Pull request by flenter**
add docs on the script step

### CODE REVIEW

**Comment by lindseybateman** [comment]
lgtm

File: content/docs/steps/script-step.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 ---
-tags: example
+tags: custom
 ---
 
 ## Script step
+
+This type of [step](/docs/steps/index.html) allows you to execute a single or
```
Comment (hatchan) [inline content/docs/steps/script-step.md]: "The `script` step allows you to ...." sounds a bit better imo.

**Comment by hatchan** [comment]
:+1:

**Comment by Tonnu** [comment]
lgtm :+1:

**Pull request merged**